package.json

```diff
@@ -52,7 +52,7 @@
     "@babel/runtime": "^7.6.3",
     "chain-function": "^1.0.1",
     "classnames": "^2.2.6",
-    "fundamental-styles": "~0.3.0-rc.4",
+    "fundamental-styles": "^0.3.0",
```

Comment:
Can you change the `^` to a `~` so we don't accidentally take breaking changes here?